helm/values.yaml

```diff
@@ -35,7 +34,12 @@ cloudProvider:
   flavor:
   imagePullSecretName:
 imagePullPolicy:
-environment:
+mcLabelsAndAnnotations:
+    environment: development
```

Comment:
Remove this line, this key should be changed in the values part and should not be with default value